app/scripts/services/authentication.js

```diff
@@ -4,43 +4,42 @@ app.service('AuthenticationService', function ($http, $q, SessionStoreService) {
   return {
     login: function(user) {
       var deferred = $q.defer();
-      $http.post('//tma-develop.herokuapp.com/v1/sessions', user)
+      $http.post('//localhost:3000/v1/sessions', user)
```

Comment:
I think the angular constant service is a good fit for this
http://twofuckingdevelopers.com/2014/06/angularjs-best-practices-001-constants/